plaid-stl/src/npm/shared_structs/mod.rs

```diff
@@ -0,0 +1,238 @@
+//! This module contains all structs which are shared between the STL and the runtime API.
```

Comment:
What's the semantic meaning of `//!`

Reply:
That's a doc comment that applies to the module itself